lib/utils.js

```diff
@@ -1,6 +1,22 @@
 // Custom Splitter:
-const split = (str, delimiter) => {
-    return (str.split(delimiter || ',').map(element => element.trim())).filter(element => element);
+const split = (str, delimiter, haswhitespace) => {
```

Comment:
If you don't mind, can you make it `hasWhitespace`?

Reply:
Done!